cairo/Cargo.toml

```diff
@@ -27,7 +27,8 @@ freetype = ["ffi/freetype", "freetype-crate"]
 script = ["ffi/script"]
 xcb = ["ffi/xcb"]
 xlib = ["ffi/xlib"]
-dox = ["ffi/dox", "glib/dox", "freetype"]
+dox = ["ffi/dox", "glib/dox", "freetype", "user-fonts"]
+user-fonts = ["ffi/user-fonts", "once_cell"]
```

Comment:
Isn't this always available in cairo? In that case it would make sense to not have a feature for that but always include it